Rakefile

```diff
@@ -71,6 +71,12 @@ namespace :spec do
       system('node --inspect-brk node_modules/.bin/jest --runInBand')
     end
   end
+
+  desc "Run integration tests"
+  task :integration do
+    system('yarn cypress:run --config video=false')
```

Comment:
Should we also disable screenshots, or does this do both?

Reply:
hehe, I couldn't find that yet....looks... hmm, looks like it's ugly right now:  https://github.com/cypress-io/cypress/issues/5029

Reply:
not sure if that works from cmdline

Reply:
Ok, I put some hacky stuff above that disables the screenshots for only this rake task... Maybe we want to do it only with ENV['CI'] or something like that.  But anyway, that appears to do it and saves some time.

Related: https://docs.cypress.io/api/cypress-api/screenshot-api.html#Arguments